manifests/init.pp

```diff
@@ -98,6 +98,32 @@
   $epel_testing_debuginfo_includepkgs     = undef,
   $epel_testing_debuginfo_sslclientkey    = undef,
   $epel_testing_debuginfo_sslclientcert   = undef,
+  $epel_playground_mirrorlist             = $epel::params::epel_playground_mirrorlist,
```

Comment:
can you please add puppet-strings like comments at the header for each parameter and add a proper datatype?